README.md

```diff
@@ -1,2 +1,65 @@
-# todoList
-This project base on the daily task list
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List (Structure)
+
+> This project base on the struction of add/remove daily task list
+
+![Screenshot (74)](https://user-images.githubusercontent.com/61976324/149188620-4fc7efe1-9122-4070-a4c6-116d9a2d6082.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages: HTML, CSS and Javascript
+- Frameworks: NO
+- Technologies used: Github and git flow, VS code
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites: Set up webpack
+
+### Setup: Vs Code 
+
+### Install: Webpack
+
+### Usage: Webpack 
+
+### Run tests: Eslint, npm
+
+### Deployment: Mode
+
```

Comment:
- [ ] [Required] We can add all the codes that are needed from cloning the repo to seeing the project in the browser here. In that case, it will be easier for anyone that may want to run the local copy of your project.

Reply:
Okay. I'll update it if I can 👌